### ISSUE TITLE
how to create the mapping?

### ISSUE THREAD

**Issue by tessus** [xdanaux/fontawesome-latex#21]
I can't figure out where the keys in the [pdftex mapping](https://github.com/xdanaux/fontawesome-latex/blob/1a6ce6ef7bbac3656095cedd86f0b99140e201d3/generate_tex_bindings.py#L22-L470) are coming from:
e.g.:

`"_620" : "percent", #f295`

The percent symbol/icon is unicode point f295, but what has _620 to do with that?

**Comment by zauguin** [comment]
Inside the OTF font file, every glyph has a name.
In FontAwesome 4 these names are highly inconsistant.
Sometimes they are the real icon names with `-` replaced by `_` like `fighter_jet`, sometimes the names are derived from the Unicode codepoint like `uniF2A0` (`phone-volume`) and some names are derived form the slot number in the font file: `_620` is the 620th icon defined in the file.

**Comment by tessus** [comment]
@zauguin thanks a lot for the answer. I guess in this case I will have to look at the otf file for [ForkAwesome](https://github.com/ForkAwesome/Fork-Awesome) and adjust the mapping accordingly.

/ref https://github.com/ForkAwesome/Fork-Awesome/issues/54

**Comment by zauguin** [comment]
@tessus I do not think this is really necessary for ForkAwesome, their icon names are much more regular and you can just parse the YAML file to create the necessary macros.

**Comment by tessus** [comment]
@zauguin good to know. this shouldn't be too hard then. cheers.
